src/rpp/rpp/observables/specific_observable.hpp

```diff
@@ -71,6 +71,8 @@ class specific_observable : public interface_observable<Type, specific_observabl
     {
         try
         {
+            // take ownership over current thread as early as possible to delay all next "current_thread" schedulings. For  example, scheduling of emissions from "just" to delay it till whole chain is subscribed and ready to listened emissions
+            const auto drain_handle = rpp::schedulers::current_thread::ensure_queue_if_no_any_owner();
```

Comment:
For everyone's education, can you put a comment with the stack iteration for the very reason of the drain here?